docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -30,11 +30,52 @@ Modeling Strategy for the Vitamin A Deficiency Cause
 
 	Describe cause in detail
 
+Cause Hierarchy
+^^^^^^^^^^^^^^^
+.. image:: vitA_cause_hierarchy.svg
+
+Health States and Sequela
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The sequela associated with the Vitamin A deficiency cause in GBD 2017 
+include moderate vision impairment loss due to Vitamin A deficiency, 
+severe vision impairment loss due to Vitamin A deficiency, blindness due to Vitamin A deficiency, 
+asymptomatic Vitamin A deficiency, Vitamin a deficiency with mild anemia, Vitamin A deficiency with moderate anemia,
+Vitamin A deficiency with severe anemia.
+
+Restrictions
+^^^^^^^^^^^^
+
+.. list-table:: GBD 2017 Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - True
+     -
+   * - YLD age group start
+     - Early Neonatal
+     - (0, 6 days], age_group_id = 2
+   * - YLD age group end
+     - 95 Plus
+     - (95, 125], age_group_id = 235
```

Comment:
The age group intervals should be [0, 7 days), [95, 125).  E.g., early neonatal includes 0 days and goes all the way up to but does not include day 7.

Reply:
Thanks for pointing it out, will fix